src/ui/screens/settings/ViewSubscriptionManagement/SubscriptionManagementItem.qml

```diff
@@ -157,71 +157,48 @@ ColumnLayout {
     }
 
     RowLayout {
-        id: relayUpsell
-        objectName: _objectName + "-relayUpsell-layout"
+        id: upgradeToAnnualSub
+        objectName: _objectName + "-upgradeToAnnualSub-layout"
 
         visible: false
         Layout.fillWidth: true
         Layout.topMargin: MZTheme.theme.listSpacing
         Layout.bottomMargin: MZTheme.theme.windowMargin
 
-        Component.onCompleted: {
-            if (visible) {
-                Glean.sample.bundleUpsellViewed.record();
-            }
-        }
-
-        ColumnLayout {
+        MZBoldInterLabel {
+            color: MZTheme.colors.purple70
+            horizontalAlignment: Text.AlignLeft
+            font.pixelSize: MZTheme.theme.fontSizeSmall
+            text: MZI18n.SubscriptionManagementUpgradeToAnnual // "Save 50%..."
+            wrapMode: Text.WordWrap
+            lineHeight: 13
+            Layout.alignment: Qt.AlignLeft
             Layout.fillWidth: true
-
-            MZInterLabel {
-                color: MZTheme.theme.fontColorDark
-                horizontalAlignment: Text.AlignLeft
-                font.pixelSize: MZTheme.theme.fontSizeSmall
-                text: MZI18n.SubscriptionManagementAddFirefoxRelay // "Add Firefox Relay"
-                wrapMode: Text.WordWrap
-                lineHeight: 13
-                Layout.alignment: Qt.AlignLeft
-                Layout.fillWidth: true
-                elide: Text.ElideRight
-            }
-
-            MZLinkButton {
-                objectName: _objectName + "-relayUpsell-learnMoreLink"
-                linkColor: MZTheme.theme.blueButton
-                fontSize: MZTheme.theme.fontSizeSmall
-                labelText: MZI18n.SplittunnelInfoLinkText // "Learn more"
-                Layout.alignment: Qt.AlignLeft
-                padding: 4
-                Layout.leftMargin: -4
-
-                onClicked: {
-                    Glean.sample.bundleUpsellLinkClicked.record();
-                    MZUrlOpener.openUrlLabel("relayPremium");
-                }
-            }
+            elide: Text.ElideRight
         }
 
+
         MZButtonBase {
```

Comment:
Would it be preferable to use an `MZButton` here?

Reply:
I don't _think_ so - but mainly because we'd have to override  a lot of baked in styling in MZButton.

Reply:
The properties of MZButton that we'd need to change would be `height`, `width`, `Layout.preferredWidth`, `Layout.preferredHeight`, the font family of the button label, and `setMargins` in MZUiStates. We also don't need the button loader.

Reply:
If you just set a `Layout.preferredWidth` and `Layout.preferredHeight` on this button instance, then `height` and `width`  will be ignored (these properties shouldn't be defined inside `MZButton` anyways)

We don't need the button loader in most of primary buttons, but it is there. It itself should be in a `Loader` (outside the scope here)

For the focus outline margins (`setMargins`), where does it say they are supposed to differ from all other primary buttons? 

On the subject of **font family** for our primary buttons - all figma mockups are using Inter Semi-Bold, whereas in the application, they are all using Metropolis Semi-Bold; [VPN-5603: Change all button labels font to Inter](https://mozilla-hub.atlassian.net/browse/VPN-5603) exists to do this work, so it being the wrong font for the time being is better, because it will at least be consistently wrong with the rest of the UI 😅 

You would have to change the button label's font pixel size (from 15 -> 13) by either exposing that property (preferred), or passing in a new `Label` for the `contentItem`

If you do decide to go the `MZButtonBase` route (which again, I don't think is the right path), we should fix:
- `font.family` - it should be `fontInterSemiBoldFamily`
-  Label `horizontalAlignment` - should be `Text.AlignHCenter
- `Accessible.name` - button label is not currently read aloud by screen readers
- I don't think the button should have `Layout.fillWidth: true` - think about if the "Save 50%" label is really short in another localization, this button will become really long
- There is a typo in the `Layout.preferredHeight` binding

Reply:
Recap of our convo for posterity:
We have moved to MZButton but this leaves us with a few unfavorable outcomes...
  - We need to stipulate `Layout.preferredWidth` to override the default. There does not appear to be a good way to do this other than to set it to `undefined`, which produces benign console complaining and feels gross. Other explored options produced polish loops, which we definitely don't want. Matt to do a quick exploration of alternatives.
  - The move to MZButton means the button label will now elide. This seems undesirable and, because we definitely don't want to make any sweeping changes in MZButton (because it's everywhere, because it can't be feature flagged), this _may_ mean a change back to MZButtonBase.